src/components/expanded-state/custom-gas/FeesPanel.tsx

```diff
@@ -752,7 +759,7 @@ export default function FeesPanel({
                 )}
                 {renderWarning(maxBaseFeeError, maxBaseFeeWarning)}
               </Box>
-              <Box marginRight="-5px">
+              <Box marginRight="-5px (Deprecated)">
```

Comment:
Same as above, this won't be a diff once https://github.com/rainbow-me/rainbow/pull/4157 is merged.